src/Module/Api/Mastodon/Lists.php

```diff
@@ -78,13 +78,16 @@ public static function post(array $parameters = [])
 
 	public static function put(array $parameters = [])
 	{
-		$data = self::getPutData();
+		$request = self::getRequest([
+			'title'          => '', // The title of the list to be updated.
+			'replies_policy' => '', // Enumerable oneOf followed list none.
```

Comment:
"followed list none"?

Reply:
It's a direct copy and paste.

Reply:
Without the original formatting it doesn't work as well.
```suggestion
			'replies_policy' => '', // Enumerable. One of: "followed", "list", or "none".
```

Reply:
Without the original formatting it doesn't work as well.
```suggestion
			'replies_policy' => '', // One of: "followed", "list", or "none".
```